src/frontend/src/pages/ProjectPage/pages/ContractPage/pages/ActiveRequestsPage/index.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import * as styles from './styles.less';
+import {
+    Button,
+    IconButton,
+    DeleteIcon,
+    EditIcon,
+    ErrorMessage,
+} from '@equinor/fusion-components';
+import PersonnelRequest from '../../../../../../models/PersonnelRequest';
+import { useAppContext } from '../../../../../../appContext';
+import SortableTable from '../components/SortableTable';
+import columns from './columns';
+
+const ActiveRequestsPage: React.FC = () => {
+    const [activeRequests, setActiveRequests] = React.useState<PersonnelRequest[]>([]);
+    const [isFetching, setIsFetching] = React.useState<boolean>(false);
+    const [error, setError] = React.useState(null);
+    const [selectedRequests, setSelectedRequests] = React.useState<PersonnelRequest[]>([]);
+    const { apiClient } = useAppContext();
+    const getRequestsAsync = async () => {
+        setIsFetching(true);
+        setError(null);
+        try {
+            const response = await apiClient.getPersonnelRequestsAsync('123', '123', "RequestState eq 0 or RequestState eq 1"); //TESTING VALUES
```

Comment:
Would like the OData part of the query to be handled by the client. The active requests page shouldn't control the query string parameters in requests made by the api client. And also the $filter should be simpler to read. E.g. 'isActive eq true'. 'RequestState eq 0 or RequestState eq 0' will mean nothing for the next developer that comes across this piece of code.

Another thing, in general about the api client: we should probably unwrap the responses within the api client instead of returning the HttpResponse<T> type 🤔 If you change the methods you've introduced in this PR, me and @olerichard will do the same